dev/com.ibm.websphere.appserver.features/visibility/public/mpTelemetry-1.1/io.openliberty.mpTelemetry-1.1.feature

```diff
@@ -26,24 +26,15 @@ IBM-API-Package: \
   io.opentelemetry.sdk.resources;type="third-party",\
   io.opentelemetry.instrumentation.annotations;type="third-party"
 -features=\
-  io.openliberty.jakarta.annotation-2.1, \
-  io.openliberty.restfulWS-3.1, \
-  io.openliberty.mpConfig-3.1, \
-  io.openliberty.cdi-4.0, \
-  io.openliberty.mpCompatible-6.1,\
-  io.openliberty.org.eclipse.microprofile.rest.client-3.0
+  io.openliberty.mpTelemetry1.1.ee-9.0; ibm.tolerates:= "9.0, 10.0"
```

Comment:
Strictly tolerates "9.0" is not needed here but I guess this line is going to change again when ee7 and ee8 support is added.